metakernel/_metakernel.py

```diff
@@ -79,6 +82,17 @@ class MetaKernel(Kernel):
     }
     meta_kernel = None
 
+    @classmethod
+    def run_as_main(cls):
+        """Launch or install a metakernel.
+
+        Modules implementing a metakernel subclass can use the following lines:
+
+            if __name__ == '__main__':
+                MetaKernelSubclass.run_as_main()
+        """
+        _MetaKernelApp.launch_instance(kernel_class=cls)
```

Comment:
But that won't support installing with `python -m$kernel_module install`, right?  What error do you get?  Works fine for me.